website/source/guides/tls.html.md

```diff
@@ -0,0 +1,375 @@
+---
+layout: "guides"
+page_title: "Securing Nomad with TLS"
+sidebar_current: "guides-tls"
+description: |-
+  Securing Nomad's cluster communication is not only important for security but
+  can even ease operations by preventing mistakes and misconfigurations. Nomad
+  optionally uses mutual TLS (mTLS) for all HTTP and RPC communication.
+---
+
+# Securing Nomad with TLS
+
+Securing Nomad's cluster communication is not only important for security but
+can even ease operations by preventing mistakes and misconfigurations. Nomad
+optionally uses mutual TLS (mTLS) for all HTTP and RPC communication. Nomad's
```

Comment:
The mutual TLS section is pretty small, doesn't use quite the same nomenclature as us, and says whether or not it's mutual is governed by the cipher suite which isn't technically correct. :(